### PR TITLE
Fix misalignment issue in repo header

### PR DESCRIPTION
**Pull request by silverwind**
Noticed a issue in the repo header where the commit date was misaligned and there was a visible issue with the bottom-border of the header. Fixed it by removing a seemingly unnecessary `vertical-alignment`.

#### Before
screen shot 2017-12-02 at 15 54 27

#### After
screen shot 2017-12-02 at 15 57 04

Edit: This is on Firefox.

### CODE REVIEW

**Comment by silverwind** [comment]
This was introduced in https://github.com/go-gitea/gitea/commit/86ee41ec033105bec50c1cc4eaf0181a37d71574. cc: @sondr3: what was the purpose of that rule?

**Comment by silverwind** [comment]
I just tested a expandable commit, looking fine with my change too:

screen shot 2017-12-02 at 16 03 34
screen shot 2017-12-02 at 16 03 25

**Comment by lunny** [comment]
LGTM

**Review by lafriks — CHANGES_REQUESTED** [review]
This will break expandable commits in code file list

**Comment by silverwind** [comment]
> code file list

Where are those? Got an example?

My test repo is at https://try.gitea.io/silverwind/longlonglonglonglonglong/.

**Comment by lafriks** [comment]
Sorry, I meant commit list - https://try.gitea.io/silverwind/longlonglonglonglonglong/commits/branch/master

**Comment by silverwind** [comment]
Ah, I see what I mean. Adjusted the selector to only apply on `tbody`, which should resolve it.

**Review by lafriks — APPROVED** [review]

**Comment by lafriks** [comment]
LGTM

**Comment by silverwind** [comment]
Commits squashed.

File: public/less/_repository.less [hunk 1]
```diff
@@ -1607,8 +1607,8 @@
 	}
 }
 
-.commit-list {
-	vertical-align: baseline;
+tbody.commit-list {
+    vertical-align: baseline;
```
Comment (lafriks) [inline public/less/_repository.less:1611]: Please fix tab back to what is in rest file

**Review by lafriks — CHANGES_REQUESTED** [review]

File: public/less/_repository.less (hunk 1 above)
Comment (silverwind) [reply public/less/_repository.less:1611]: But, that would contradict [`.editorconfig`](https://github.com/go-gitea/gitea/blob/35cc5b0402d46d672e02bbe1ad15d1460077e8f4/.editorconfig#L18-L20). See https://github.com/go-gitea/gitea/pull/3064 for fixing the `.less` files indendation issues.

Comment (silverwind) [reply public/less/_repository.less:1611]: Also, the upper half of that file is already intended using spaces, btw :)

**Review by lafriks — APPROVED** [review]

**Comment by lafriks** [comment]
Oh well, ok :)

**Comment by sondr3** [comment]
I didn’t check in Firefox and it looked fine in Chrome so I didn’t think to check, you can see in my pull request that it aligns properly with my original code. CSS is weird.

**Comment by silverwind** [comment]
Yes, people often forget to test other browers these day. Generally I'd say "old" methods like tables are weird. Would probably not happened with Flexbox, for example. :)

**Pull request merged**